### PR TITLE
Update kf-deployment-ui-version.html

### PR DESCRIPTION
**Pull request by rky0930**
Now KF v0.7.0 can be deployed using UI.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1458)

### CODE REVIEW

File: layouts/shortcodes/kf-deployment-ui-version.html [hunk 1]
```diff
@@ -1 +1 @@
-v0.6.2
\ No newline at end of file
+v0.7.0
```
Comment (sarahmaddox) [inline layouts/shortcodes/kf-deployment-ui-version.html]: Remove the new line at end of file.

We're using Hugo shortcodes as inline variables, and a new line in the shortcode breaks the formatting. For example, see the preview generated for your PR: https://deploy-preview-1458--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/

**Comment by sarahmaddox** [comment]
Hallo @rky0930 a hint in case you're using the GitHub UI to make this change: Unfortunately, the GitHub UI always puts the new line back. You need to use the command line. You can find a quick guide here: https://github.com/kubeflow/website/blob/master/quick-github-guide.md#using-the-command-line

**Comment by sarahmaddox** [comment]
Thanks!
/lgtm
/approve

**Comment by rky0930** [comment]
Thank you for helping me. I didn't know github UI always puts the new line back.
I remove the newline at end of file using cli and checked deploy preview. It seems OK now. 
If you find anything else, I will contribute again.

**Pull request merged**